2020/better-bayes/russian/sentence_translations.json

```diff
@@ -469,7 +469,7 @@
  },
  {
   "input": "We might call this a veridical paradox, which refers to facts that are provably true, but which nevertheless can feel false when phrased a certain way. ",
-  "translatedText": "Мы могли бы назвать это правдивым парадоксом, который относится к фактам, которые доказуемо истинны, но которые, тем не менее, могут показаться ложными, если их сформулировать определенным образом. ",
+  "translatedText": "Мы могли бы назвать это таким парадоксом, который говорит о фактах, которые доказуемо истинны, но, тем не менее, могут показаться ложными, если их представить определенным образом. ",
```

Comment:
I have not found any good translation for «veridical paradox» in russian literature, so I've changed it to something like «we might call such paradox as which refers to facts …» removing the term completely.